sklearn/utils/validation.py

```diff
@@ -2169,17 +2177,18 @@ def _check_sample_weight(
         Validated sample weight. It is guaranteed to be "C" contiguous.
```

Comment:
Just checking, is this the same changes as made in #30878? Since that one as the additional tests, maybe it should be merged first?